### ISSUE TITLE
VS Code shouldn't open getting started docs on first start

### ISSUE THREAD

**Issue by oliversalzburg** [Microsoft/vscode#49066]
Issue Type: <b>Bug</b>

When you download VS Code, you are redirected to, for example, https://code.visualstudio.com/docs/?dv=win

You then install VS Code and open it and it pops open the browser to send you to https://code.visualstudio.com/docs?start=true, the page you just came from. 

Don't pop open my browser to distract me from what I originally wanted to do, just to show me a page I just saw a minute ago. Why not use the same in-application functionality that powers the Release Notes after an update?

VS Code version: Code 1.22.2 (3aeede733d9a3098f7b4bdc1f66b63b0f48c1ef9, 2018-04-12T16:38:45.278Z)
OS version: Windows_NT x64 10.0.16299

<details>
<summary>System Info</summary>

|Item|Value|
|---|---|
|CPUs|Intel(R) Core(TM) i7-6850K CPU @ 3.60GHz (12 x 3672)|
|Memory (System)|63.91GB (52.71GB free)|
|Process Argv|C:\Program Files\Microsoft VS Code\Code.exe|
|Screen Reader|no|
|VM|0%|

</details>Extensions: none

**Comment by auchenberg** [comment]
Thanks for your feedback @oliversalzburg. We have had this current behavior for quite some time, and had positive feedback from our users, as we only open our getting started guide when you install VS Code for the very first time.

I'm keeping this issue open for now to see if more users have similar feedback.

**Comment by oliversalzburg** [comment]
Given that this is a one-time thing after a fresh installation, I wouldn't expect many people to give feedback on this as they probably get over it very quickly (if they even notice at all that they now have the same page open twice).

It's just obviously redundant, as the site provides no additional value at that point and it just distracts me from what I originally wanted to do - start VS Code. And there is *so much* good stuff on the Welcome page already, why send me to an external application, to a page I just came from when I downloaded the application?

I feel like *not* opening the page would be an improvement to the onboarding procedure. Just wanted to elaborate a bit, as I wanted to keep the original ticket brief.
